Gemfile

```diff
@@ -95,11 +67,94 @@ gem "ruport",                         "=1.7.0",                       :git => "h
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
 
+### providers
+group :amazon do
+  manageiq_plugin "manageiq-providers-amazon"
+end
+
+group :ansible do
+  gem "ansible_tower_client",           "~>0.12.2",      :require => false
+end
+
+group :azure do
+  manageiq_plugin "manageiq-providers-azure"
+end
+
+group :foreman do
+  gem "foreman_api_client",             ">=0.1.0",   :require => false, :git => "https://github.com/ManageIQ/foreman_api_client.git", :branch => "master"
+end
+
+group :google do
+  gem "fog-google",                     ">=0.5.2",       :require => false
+  gem "google-api-client",              "~>0.8.6",       :require => false
+  gem "omniauth",                       "~>1.3.1",       :require => false
+  gem "omniauth-google-oauth2",         "~>0.2.6",       :require => false
+end
+
+group :hawkular do
+  manageiq_plugin "manageiq-providers-hawkular"
+end
+
+group :kubernetes do
```

Comment:
Since `manageiq-providers-openshift` pretty much inherits from `manageiq-providers-kubernetes`, you might just want to make this `group :kubernetes, :openshift do`.